bitcash/format.py

```diff
@@ -10,11 +10,19 @@
 MAIN_PRIVATE_KEY = b'\x80'
 MAIN_BIP32_PUBKEY = b'\x04\x88\xb2\x1e'
 MAIN_BIP32_PRIVKEY = b'\x04\x88\xad\xe4'
+
 TEST_PUBKEY_HASH = b'\x6f'
 TEST_SCRIPT_HASH = b'\xc4'
 TEST_PRIVATE_KEY = b'\xef'
 TEST_BIP32_PUBKEY = b'\x045\x87\xcf'
 TEST_BIP32_PRIVKEY = b'\x045\x83\x94'
+
+REGTEST_PUBKEY_HASH = b'\x6f'
+REGTEST_SCRIPT_HASH = b'\xc4'
+REGTEST_PRIVATE_KEY = b'\xef'
+REGTEST_BIP32_PUBKEY = b'\x045\x87\xcf'
+REGTEST_BIP32_PRIVKEY = b'\x045\x83\x94'
+
```

Comment:
If these will always be the same, let's do:
```python
REGTEST_PUBKEY_HASH = TEST_PUBKEY_HASH
```
and so on?

Reply:
Agreed.

Reply:
Sounds good to me. I made the change (see: 2e18d4d)